packages/mediapipe-task-text/pubspec.yaml

```diff
@@ -17,14 +25,13 @@ dependencies:
     sdk: flutter
   http: ^1.1.0
   logging: ^1.2.0
-  mediapipe_core:
-    path: ../mediapipe-core
+  mediapipe_core: ^0.0.1
   native_assets_cli: ^0.3.0
   native_toolchain_c: ^0.3.0
   path: ^1.8.3
 
 dev_dependencies:
   ffigen: ^9.0.1
-  flutter_lints: ^2.0.0
```

Comment:
DBC: If I'm not mistaken the list of flutter lints is bigger than the lints package. So you might as well keep flutter_lints if you don't mind satisfying them. It makes the code more pleasant to read to typical flutter users.